test/makeRequest.spec.js

```diff
@@ -1,68 +0,0 @@
-'use strict';
-
-require('../testUtils');
-
-const makeRequest = require('../lib/makeRequest');
-const utils = require('../lib/utils');
-const expect = require('chai').expect;
-
-describe('makeRequest', () => {
-  describe('args', () => {
-    it('does not mutate user-supplied deprecated opts', () => {
-      const args = [
-        {
-          stripe_account: 'bad',
-        },
-      ];
-      const mockSelf = {
-        createResourcePathWithSymbols: () => {},
-        createFullPath: () => {},
-        _request: () => {},
-      };
-      makeRequest(mockSelf, args, {}, {});
-      expect(args).to.deep.equal([
-        {
-          stripe_account: 'bad',
-        },
-      ]);
-    });
-  });
-
-  describe('makeRequest with fullPath', () => {
```

Comment:
Do we still want some of these tests in RequestSender.spec.js?

Reply:
I copied the `does not mutate user-supplied deprecated opts` to StripeResource tests. 

The `makeRequest with fullPath` is covered by `method with fullPath -> interpolates in parameters`.